website/src/hooks/useIsPage.ts

```diff
@@ -0,0 +1,12 @@
+import { usePathname } from 'next/navigation';
+
+export function useIsPage(page: string) {
```

Comment:
This hook seems to have a quite specific use-case. It only checks for the third item in the path segments. Maybe a hook `usePathnameContains(page: string)` would be more re-usable?

You then still do 
```ts
const isSurveyPage = usePathnameContains('/[lang]/[region]/survey');
```

What do you think?

Reply:
That’s a good idea, I’ll update it.